.github/workflows/run-checker-ci.yml

```diff
@@ -25,7 +28,7 @@ jobs:
           no-ec,
           no-ecx,
           no-http,
-          no-legacy,
+          no-legacy enable-unstable-qlog,
```

Comment:
Why in combination with no-legacy?

Reply:
It was randomly chosen as an existing CI item to piggyback on, rather than adding yet *another* CI job.